res/openvpn-gui-res-no.rc

```diff
@@ -244,6 +246,12 @@ BEGIN
 
     /* OpenVPN */
     IDS_ERR_MANY_CONFIGS "Du kan ikke ha flere enn %d konfigurasjonsfiler. Kontakt utvikleren av OpenVPN GUI om du har behov for å håndtere flere filer."
+    IDS_NFO_NO_CONFIGS "Ingen lesbare tilkoblingsprofiler (konfigurasjonsfiler) funnet"
```

Comment:
Introduced a new term here "tilkoblingsprofiler". Should probably either be used more places as well, or not used at all. The English translation reads "No readable connection profiles (config files) found"